packages/ui/src/components/popover/popover.vue

```diff
@@ -43,7 +43,9 @@
         class="acv-popover"
       >
         <AcvCard
-          class="acv-popover__content"
+          class="content"
+          background-color="white"
```

Comment:
Why strictly white? shouldn't there be a dependency on brand or light/dark mode?

Reply:
Now we don't have dark variation. 
Its just white in Acronis ui-kit.
New mockups also don't have dark scheme.